dev/com.ibm.ws.microprofile.config.1.2_fat/test-applications/converterApp/src/com/ibm/ws/microprofile/config12/converter/type/web/TypeConverterBean.java

```diff
@@ -128,7 +128,9 @@ public void setConverterTest() throws Exception {
      */
     public void optionalConverterTest() throws Exception {
         assertNotNull(unknownOptional);
-        assertFalse(unknownOptional.isPresent());
+        if (unknownOptional.isPresent()) {
+            fail("optional value should have been null but was (MyStringObject): " + unknownOptional.get().getValue());
+        }
```

Comment:
Much better, though we should write a matcher if we do this sort of thing a lot.

Reply:
feel free :-)

Reply:
That comment was more to the effect of "if you're doing this a lot, be aware there's a better way". 

I'm happy to write one if it would be helpful.